bundles/test/org.eclipse.smarthome.test/src/main/java/org/eclipse/smarthome/test/OSGiTest.java

```diff
@@ -51,6 +51,7 @@
  * @author Dennis Nobel - Initial contribution
  * @author Tanya Georgieva - Refactor the groovy file to java
  */
+@Deprecated
```

Comment:
I assume we can deprecate the whole class and keep it as long as it used by ESH products. Or should it be moved to the products itself as it is not used by ESH itself anymore. This will allow us to drop Groovy from p2 etc.

Reply:
We can check this with DT. The groovy base class is used there.